src/Flow.cpp

```diff
@@ -0,0 +1,19 @@
+#include "Flow.hpp"
+#include "NetworkSimulator.hpp"
+
+#include <stdexcept>
+
+Flow::Flow(const std::vector<std::shared_ptr<NetworkDevice>>& path,
```

Comment:
Sorry that I confused you here with the flow paths. As soon as we now have a routing table in the PR https://github.com/Cloud-Storage-Team/algnet/pull/33, I think, there is no need to set path in the constructor anymore, but only *source* and *destination*. Basically, the flow should be a direct analog of the *network socket pair*. Creating/deleting flows is equivalent to opening/closing a socket.